advanced_survey/static/advanced-survey/build/direflowBundle.js

```diff
@@ -0,0 +1,3 @@
+/*! For license information please see direflowBundle.js.LICENSE.txt */
```

Comment:
Can you confirm this is meant to be committed?

Reply:
I think so because it used in the edit form page